app/server/endpoint/src/main/java/io/syndesis/server/endpoint/v1/handler/connection/ConnectorHandler.java

```diff
@@ -129,7 +127,7 @@ public Connector get(final String id) {
 
         // Retrieve dynamic properties, if connector is dynamic
         if (connector.getTags().contains("dynamic")) {
-            connector = enrichWithDynamicProperties(connector);
+            connector = enrichConnectorWithDynamicProperties(connector);
```

Comment:
I'd remove this from the `GET /connectors/{id}`. I'd like the retrieval of dynamic metadata, i.e. invoking of the syndesis-meta service to be tied to an explicit request from the UI for it, not in every case when the connector is fetched.

`GET` requests should be idempotent and should not have side effects, this will endanger that.

Reply:
I agree, though it would be out the scope of the request for this PR (circuit breaker). I have created  further issue to take care of this refactory #8023 separately. Are you okey to proceed like this?

Also, from performance perspective, right now the "enriching" call is conditioned to the presence of a tag that we are configuring on the connectors which use this feature.